controlplane/modules/route/cfg.go

```diff
@@ -8,7 +13,19 @@ type Config struct {
 	MemoryPathPrefix string `yaml:"memory_path_prefix"`
 	// MemoryRequirements is the amount of memory that is required for a single
 	// transaction.
-	MemoryRequirements uint   `yaml:"memory_requirements"`
-	Endpoint           string `yaml:"endpoint"`
-	GatewayEndpoint    string `yaml:"gateway_endpoint"`
+	MemoryRequirements datasize.ByteSize `yaml:"memory_requirements"`
+	Endpoint           string            `yaml:"endpoint"`
+	GatewayEndpoint    string            `yaml:"gateway_endpoint"`
+	// BirdExport configures the reader for the Bird Export Protocol feed.
+	BirdExport *bird.Config `yaml:"bird_export"`
```

Comment:
This is optional field, right? What happens if we omit `bird_export` in the config?

Reply:
By default, nothing happens; instead, an informative log message will be printed to the console.
```
m.log.Info("bird export reader is disabled")
```